src/xdocs/config_coding.xml

```diff
@@ -2291,6 +2298,17 @@ public native void myTest(); // violation
   &lt;property name=&quot;ignoreCase&quot; value=&quot;true&quot;/&gt;
 &lt;/module&gt;
         </source>
+          <p>Example:</p>
+          <div class="wrapper">
+              <pre>
```

Comment:
Same as above

Reply:
ok, I will fix it.

Reply:
I fix it and update the first post~

Reply:
@HuGanghui please reply all my comments with `done` if you fixed them